packages/tailwindcss-language-server/src/matching.ts

```diff
@@ -0,0 +1,24 @@
+import picomatch from 'picomatch'
+import { DefaultMap } from './util/default-map'
+
+export interface PathMatcher {
+  anyMatches(pattern: string, paths: string[]): boolean
+  clear(): void
+}
+
+export function createPathMatcher(): PathMatcher {
+  let matchers = new DefaultMap<string, picomatch.Matcher>((pattern) => {
```

Comment:
`DefaultMap` coming in clutch and saving our buds once again

Reply:
FOR REAL